k8s/aat/common/adoption/adoption-backend.yaml

```diff
@@ -24,6 +24,8 @@ spec:
       prometheus:
         enabled: true
       aadIdentityName: adoption
+      environment:
+        CORE_CASE_DATA_API_URL: http://ccd-data-store-api-aat.service.core-compute-aat.internal
```

Comment:
should this not be defined in aat.tfvars?

Reply:
Infrastructure code should have any app specific vars.